README.md

```diff
@@ -1,2 +1,50 @@
-# PLuginCiscoACI
-Implement an open source plugin to manage Cisco ACI fabrics
+# PluginCiscoACI
+PluginCiscoACI
+This repository hosts the code for HPE ACI Plugin
+
+## Docker Container Install
+This makes it easier to stand up aci-plugin. You do with a simple "make all" and you will get a aci-plugin container running with a port open on 45003
+### Required Sofware to Install
+
+To be able to install with docker you need:
+* docker-ce
+* docker-ce-cli
+* docker-compose
+* golang-docker-credential-helpers
+* python-docker
```

Comment:
Are these the same that are used in upstream?. Anyway if we move to kubernetes these wont be needed. Isnt it ?

Reply:
yes @Bharath-KKB this is the same as what we have in upstream, and it will be required for docker deployment. But for Kubernetes deployment, this is not required.